connector/openshift/openshift.go

```diff
@@ -246,16 +249,23 @@ func validateAllowedGroups(userGroups, allowedGroups []string) bool {
 }
 
 // newHTTPClient returns a new HTTP client
-func newHTTPClient(insecureCA bool, rootCA string) (*http.Client, error) {
+func newHTTPClient(insecureCA bool, includeHostRootCA bool, rootCA string) (*http.Client, error) {
```

Comment:
Very minor, but I would move the `bool` parameter last. That's because it only has a meaning if `rootCA` is set/non-empty (otherwise it's the default), so there is some dependency.